docs/manuals/source/DeveloperGuide/release.rst

```diff
@@ -0,0 +1,74 @@
+Releasing Bareos
```

Comment:
How about:

```suggestion
Preparing a Bareos Release
```

Or is it planed to extend this document to fully cover the release process?

Reply:
What exactly is missing for the release of the source-code?

Reply:
Your next 3 sentences describe pretty clear and well what this section covers.
However, my first expectation when I'm reading the header "Releasing Bareos" is that a "Release" would also cover building and especially distributing. I tried I find a heading that better covers the content of this section.

To answer your question: I think this sections covers all aspects of the Bareos source-code release. So nothing is missing for this.

Reply:
As you mention below, that creating a tar-ball is not part of "Releasing Bareos" (this section): This contradict somehow with the github definition of a release. On github, a tar.gz is created for every published tag: https://github.com/bareos/bareos/releases

If I understood correctly, these can not (easily) be used to manually create binary packages. I think, this should be covered or at least mentioned in the introduction.

Reply:
In fact GitHub even allows you to download a .tar.gz or .zip for each and every commit. So yes, we should definitely describe this somewhere (maybe in the how to build documentation?).

Reply:
I think it would be best to document the version number generation and its requirements and then link to that when cmake fails because of a problem relating to that.
Don't think we have another option to tell every existing and future user how to fix this if it breaks.

Reply:
The improved error handling for the version generation will be implemented by #362

Reply:
Also the automation-script now takes care that at least the release-tagged tarballs can be built. The version number is injected for the release-commit and will be removed after the release-commit with another commit.